docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -343,6 +343,7 @@ statements, the connector supports the following features:
 * :doc:`/sql/drop-schema`
 * :doc:`/sql/show-create-schema`
 * :doc:`/sql/show-create-table`
+* :doc:`/sql/comment`
```

Comment:
Note that Iceberg & Hive supports  also `COMMENT`.
(Mongo is already documented)

Reply:
@colebow can check and send PRs